### DISCUSSION TITLE
RIA_OP API - Doc (flipped RIA_A / RIA_X) and include-files def (RIA_OP_CLOCK)

### DISCUSSION THREAD

**Discussion by netzerohero** [picocomputer/community#81] [Q&A]
1. I think for the RIA Registers documentation, RIA_A and RIA_X are flipped for the table in Section 2.2:
![RIA_A_X_flipped (corrected)](https://github.com/picocomputer/community/assets/148929770/7b0a2e97-c310-4024-9d69-4754e9325888)

2. In my exploring of the RIA_OP_CLOCK operation with the recent clock() api addition, in the cc65 repo files: (a)  asminc/**rp6502.inc**  may be missing the RIA_OP_CLOCK definition ($0F)  that is in (b) include/**rp6502.h**.  (RIA_OP_CLOCK was added after the current 0.6 firmware release).

**Comment by rumbledethumps** [comment]
Fixed. Thanks.
